lnwire/lnwire.go

```diff
@@ -515,6 +524,18 @@ func readElement(r io.Reader, element interface{}) error {
 		}
 
 		*e = newAlias(a[:])
+	case *DeliveryAddress:
+		var addrLen [2]byte
+		if _, err = io.ReadFull(r, addrLen[:]); err != nil {
+			return err
+		}
+		length := binary.BigEndian.Uint16(addrLen[:])
+
+		var addrBytes [32]byte
```

Comment:
The max value here is actually _34 bytes_. 34 bytes are needed to encode the script for a p2wsh script: `OP_0 OP_PUSH_DATA_32 <32 bytes>`.